.github/workflows/tests.yml

```diff
@@ -251,7 +239,17 @@ jobs:
           set PERL=
           set LUA=
           set R=
-          pytest --color=yes -vv -n 0 --basetemp %UserProfile%\cbtmp_serial --cov conda_build --cov-report xml -m "serial" ${{ env.pytest-replay }}
+          pytest ^
```

Comment:
24-caret

What is the ^ character doing at the end of these lines?

Reply:
that's the line continuation character for CMD